packages/e2e/src/page-objects/messages.page.ts

```diff
@@ -19,23 +19,220 @@ class MessagesPage extends Page {
+  > {
+    return $('[data-testid="message-bubble-message-content"]')
+  }
+
+  async hoverOverMessage() {
+    await this.containerMessageContent.waitForDisplayed({ timeout: 9000 })
+    await this.containerMessageContent.moveTo()
+  }
+
+  public get messageOptionsDropdownList(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="message-bubble-dropdown-action-button"]')
+  }
+
+  //
+  public get optionsButtonSingleMessage(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return this.messageContainer.$('[data-testid="icon-More"]')
+  }
+
+  async clickSingleMessageOptionsButton() {
+    await this.optionsButtonSingleMessage.waitForClickable()
+    await this.optionsButtonSingleMessage.click()
+  }
+
+  public get threadScreenEmptyList(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="messages-empty-thread-list-state"]')
+  }
+
+  public get threadScreenThreadDropdownOptions(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="dropdown"]')
+  }
+
+  public get deleteOnThreadDropdownOptions(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return this.threadScreenThreadDropdownOptions.$(
+      '[data-testid="message-bubble-delete-message-button"]'
+    )
+  }
+
+  async clickDeleteMessageOnThreadScreen() {
+    await this.deleteOnThreadDropdownOptions.waitForClickable()
+    await this.deleteOnThreadDropdownOptions.click()
+  }
+
+  public get buttonOptionsThreadListScreen(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="icon-More"]')
+  }
+
+  public get optionDeleteDropdown(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="dropdown-delete"]')
+  }
+
+  async clickNewMessageButton() {
+    await this.newMessageButton.waitForClickable({ timeout: 7000 })
+    await this.newMessageButton.click()
+  }
+
+  async insertTextToSearchContactInput(phoneNumber: string) {
+    await this.searchContactsInput.waitForDisplayed()
+    await this.searchContactsInput.setValue(phoneNumber)
+  }
+
+  async insertTextToMessageInput(messageText: string) {
+    await this.messageTextInput.waitForDisplayed()
+    await this.messageTextInput.setValue(messageText)
+  }
+
+  async sendMessage(phoneNumber: string, messageText: string) {
+    // click NEW MESSAGE button
+    await this.clickNewMessageButton()
+    //input phone number into contact search field
+    await this.insertTextToSearchContactInput(phoneNumber)
+    //input message text into message input
+    await this.insertTextToMessageInput(messageText)
+    //send by pressing Send button
+
+    await this.sendMessageButton.waitForClickable()
+    await this.sendMessageButton.click()
+  }
+
+  async clickCloseThreadDetailsScreen() {
+    await this.closeButtonOnThreadDetailScreen.waitForClickable({
+      timeout: 7000,
```

Comment:
🟢 I noticed that timeout is usually set to 7000 ms. You might want to move the timeout value to the dedicated const.